gum/backend-darwin/gumprocess-darwin.c

```diff
@@ -1380,6 +1474,51 @@ gum_darwin_enumerate_ranges (mach_port_t task,
   }
 }
 
+static gboolean
+gum_darwin_fill_file_mapping (gint pid,
+                              mach_vm_address_t address,
+                              GumFileMapping * file,
+                              struct proc_regionwithpathinfo * reginfo)
+{
+  gint retval, len;
+
+  retval = __proc_info (PROC_INFO_CALL_PIDINFO, pid, PROC_PIDREGIONPATHINFO,
+            (uint64_t) address,  reginfo, sizeof (struct proc_regionwithpathinfo));
```

Comment:
- Exceeds 80 cols.
- Hanging indent should be 4 spaces. (Which probably solves the 80 cols issue.)